src/familiar.ts

```diff
@@ -222,10 +224,37 @@ export function pocketProfessorLectures(): number {
   return 2 + Math.ceil(Math.sqrt(familiarWeight($familiar`Pocket Professor`) + weightAdjustment()));
 }
 
-function timeToMeatify(): boolean {
-  return (
-    have($familiar`Grey Goose`) &&
-    $familiar`Grey Goose`.experience >= 400 &&
-    !get("_meatifyMatterUsed")
-  );
+export function timeToMeatify(): boolean {
+  if (!have($familiar`Grey Goose`) || get("_meatifyMatterUsed") || myInebriety() > inebrietyLimit()) return false;
+  else if ($familiar`Grey Goose`.experience >= 400) return true;
+  else if (!globalOptions.ascending || myAdventures() > 50) return false;
+
+  // Check Wanderers
+  // const F = get("_sausageFights");
+  const totalTurns = totalTurnsPlayed();
+  const baseMeat = (have($item`SongBoom™ BoomBox`)) ? 275 : 250;
+  const usingLatte = (have($item`latte lovers member's mug`) && get("latteModifier").split(",").includes("Meat Drop\: 40")) ? true : false;
```

Comment:
You should basically never have to do `x ? true : false`. You might even be able to import usingLatte from Outfit.ts
My bad for not mentioning it.

Reply:
When you put it like that it does indeed look silly, but I intentionally plucked it out for readability because the conditional was extremely lengthy